pkg/services/accesscontrol/evaluator/evaluator_test.go

```diff
@@ -0,0 +1,65 @@
+package evaluator
+
+import (
+	"testing"
+
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExtractPermission(t *testing.T) {
+	targetPermission := "permissions:create"
```

Comment:
Better make this non-variable:
```suggestion
	const targetPermission = "permissions:create"
```